src/managers/PlayerManager.js

```diff
@@ -1,20 +1,58 @@
 const Player = require('../structures/Player');
 const ReqUtil = require('../util/ReqUtil');
-
+const CacheManager = require('./CacheManager');
 class PlayerManager {
     constructor(client){
         this.client = client;
+
+        this.cache = new CacheManager(client, Player);
```

Comment:
I assume you're using underscores like `_value` to denote private members, right? I'd definitely do the same thing for the cache - I don't really want users to interact with it directly.

Reply:
Fixed